04-spatial-operations.Rmd

```diff
@@ -26,17 +26,22 @@ library(spData)
 Spatial operations are a vital part of geocomputation.
 This chapter shows how spatial objects can be modified in a multitude of ways based on their location and shape.
 The content builds on the previous chapter because many spatial operations have a non-spatial (attribute) equivalent.
-Spatial operations on *vector* objects include spatial subsetting (covered in section \@ref(spatial-subsetting)) and spatial joining (section \@ref(spatial-joining)).
-This content builds on section \@ref(vector-attribute-manipulation) in the previous chapter.
-Spatial operations on *rasters* include merging and subsetting, covered in section \@ref(spatial-ras).
-
-The chapter also introduces new concepts that are unique to spatial data.
-A variety of *topological relations* can be used to subset/join vector geometries (by default **sf** uses the catch-all *intersects* but other relations such as *within* can be very useful), a topic that is explored in section \@ref(topological-relations).
-Spatial operations on raster datasets involve *map algebra* (covered in sections \@ref(map-algebra) to \@ref(global-operations-and-distances)) and raster merging (covered in sections \@ref(merging-rasters)).
-
+This is especially true for *vector* operations.
+For instance, we will extend the basics on vector attribute manipulation (covered in section \@ref(vector-attribute-manipulation)) with its spatial counterpart, namely spatial subsetting (covered in section \@ref(spatial-subsetting)) and spatial joining (section \@ref(spatial-joining)).
+Imagine you want to add an attribute of a road (such as the road number or the speed limit) to a municipality polygon.
+Equally, you might want to select all roads of a specific municipality.
+Were we in possession of an ID this would simply repeat the stuff introduced in the previous chapter. 
```

Comment:
"simply repeat the stuff" sounds informal. I've reworded it.

Reply:
You are right, thanks.